doc/ug/otbn_sw.md

```diff
@@ -23,19 +23,19 @@ For specific formatting and secure coding guidelines, see the [OTBN style guide]
 
 ### Assembler
 
-The OTBN assembler is called `otbn-as` and can be found at `hw/ip/otbn/util/otbn-as`.
+The OTBN assembler is called `otbn_as.py` and can be found at `hw/ip/otbn/util/otbn_as.py`.
```

Comment:
It could be argued that the assembler is now called `//hw/ip/otbn/util:otbn_as` (and similarly for the linker).

Reply:
That's what its name morally always was, only it was not hermetic. :)

Hopefully in the fullness of time we can eliminate almost all shebangs in the repo.